tutorials/terraform/sqoop/clusters-mysql-data-proc-and-vm.tf

```diff
@@ -0,0 +1,254 @@
+# Infrastructure for the Yandex Cloud Managed Service for MySQL® cluster.
+#
+# RU: https://cloud.yandex.ru/docs/managed-mysql/tutorials/sqoop
+#
+# Set the configuration of the Managed Service for MySQL® cluster, Managed Service for Data Proc cluster and Virtual machine:
+locals {
+  folder_id           = ""      # Your folder ID.
+  network_id          = ""      # Network ID for Managed Service for MySQL® cluster, Data Proc cluster and VM.
```

Comment:
Network ID for the Managed Service for MySQL® cluster, Data Proc cluster, and VM